src/sparql/keywords.ts

```diff
@@ -1,626 +1,152 @@
-// @ts-ignore: import types for declarations
-import { createToken, TokenType } from 'chevrotain';
-
-const MAX_LENGTH = createToken({
+import { ITokenConfig } from 'chevrotain';
+import helper from 'helpers/chevrotain';
+
+export const createKeyword = ({
+  longer_alt = UNKNOWN,
+  ...props
+}: ITokenConfig) =>
+  helper.createKeyword({
+    longer_alt,
+    ...props,
+  });
+
+const UNKNOWN = createKeyword({ name: 'UNKNOWN', pattern: /\w+/i });
```

Comment:
Unknown isn't technically a keyword, so I don't believe it makes sense to use the create keyword constructor.

Reply:
yes, not really sure what a keyword is. Though it would seem like they are a fall threw class of keywords.

Reply:
Kind of like a programming keyword, it's a word or collection of words that is reserved to have a special meaning or function. Something that is then identified as `unknown` cannot be a keyword since by definition, a keyword is explicitly defined and identified as part of the language.